docs/guides/references/troubleshooting.mdx

```diff
@@ -247,34 +247,69 @@ an issue you have by clearing this app data.
 
 ## Print DEBUG logs
 
-Cypress is built using the [debug](https://github.com/visionmedia/debug) module.
-That means you can receive helpful debugging output by running Cypress with this
-turned on. **Note:** you will see a LOT of messages when running with
+Cypress is built using the [debug](https://github.com/debug-js/debug) module.
+That means you can receive helpful debugging output by running Cypress with
+the environment variable `DEBUG` set to `cypress:*` prior to running `cypress run` or `cypress open`.
```

Comment:
People have gotten confused with system env vars and Cypress env vars in the past.

```suggestion
That means you can receive helpful debugging output by running Cypress with a system-level environment variable `DEBUG` set to `cypress:*` prior to running `cypress run` or `cypress open`.
```